boards/iotlab-m3/include/periph_conf.h

```diff
@@ -43,6 +43,25 @@ extern "C" {
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (1U)
+#define ADC_0_EN            1
+#define ADC_MAX_CHANNELS    17
+
+/* ADC 0 configuration */
+#define ADC_0_DEV           ADC1
+#define ADC_0_CHANNELS      17
```

Comment:
I think as long as we just have the ability to use one adc pin on the iotlab-m3 board we should define that macro with `1`